doc/about.md

```diff
@@ -0,0 +1,34 @@
+## Setting about page
+
+The About page is set automatically based on the information provided in configuration. It shows the information about app name, app version and app maintainers.
+
+```
+let ConfigDefaults = {
+	title: "LogMan.io",
+	version: "1.0",
+	maintainer: "TeskaLabs",
+	email: "info@teskalabs.com",
```

Comment:
Add also a date of release (YYYY-MM-DD)

Reply:
what date should i pass for Logman?

Reply:
should i add an additional Row-element for "Date of release"?

Reply:
Yes, it should be a new row for the date of release and I suppose it is a mandatory value, since it is related to the version.
So the example can be e.g.

`release_date: "2020-01-21"`

Reply:
👍